openstack/cbr/v3/cbrtags/requests.go

```diff
@@ -1,15 +1,14 @@
-package tags
+package cbrtags
 
 import (
 	"fmt"
 
 	golangsdk "github.com/opentelekomcloud/gophertelekomcloud"
-	"github.com/opentelekomcloud/gophertelekomcloud/openstack/cbr/v3/vaults"
```

Comment:
are you sure that all works with common tags?

Reply:
Yes, please check the OpenStack API. All of the tags operate should invoke with this func.
as I said https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/issues/1598#issuecomment-1191995932